app/src/main/java/com/github/fi3te/notificationcron/data/local/NotificationCronInput.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.fi3te.notificationcron.data.local
+
+data class NotificationCronInput (
+        val cron: String,
+        val notificationTitle: String,
+        val notificationText: String,
+        val onClickURI: String
```

Comment:
For a better readability I would prefer onClickUri

Reply:
Done!